server/resources/translations/common.en.json

```diff
@@ -9,6 +10,10 @@
   "incidentCategorized": "The incident was categorized as {{ incidentCategory }}.\n\nThank you. This session is now complete. (You don't need to respond to this message.)",
   "incidentCategoryInvalid": "Sorry, the incident type wasn't recognized. Please try again.",
   "incidentCategoryRequest": "Once you have responded, please reply with the number that best describes the incident:\n{{ incidentCategories }}",
+  "resetNoticeToOtherResponders": "Another Responder has acknowledged this request and decided to reset the Brave Sensor. (You don't need to respond to this message.)",
+  "resetRequestAccepted": "This Brave Sensor will be reset, and you will no longer receive alerts for this session.\n\nPlease note that after a reset, the sensor will not accurately monitor for stillness until about ten minutes have passed. If you have not checked on this location yet, please do so.\n\nThis session is now complete. (You don't need to respond to this message.)",
```

Comment:
I feel like "not accurately monitor" sounds like something's gone wrong and it's our fault. 

I think I prefer Gordon's suggestion of "it takes approximately 10 minutes for the system to be fully functional again." because it sounds more like just the natural consequence of a reset.

What do you think?

Reply:
Added!

Reply:
@micahdbak , I think that you might have made a change to address one of my other comments, but missed this one.

Reply:
I see that you're going to check the text with the rest of the team, but I strongly feel that "not accurately monitor" makes us sound bad.

Reply:
OK; I'll change this!

Reply:
I've chosen to replace the phrase "Please note that after a reset, the sensor will not accurately monitor for stillnesss..." with "The sensor will return to normal operation after about ten minutes have passed."